zipkin-server/README.md

```diff
@@ -7,7 +7,11 @@ the HTTP collector (POST /api/v2/spans endpoint) is enabled, and the server list
 Zipkin Server is implemented with [Armeria](https://github.com/line/armeria). While it uses [Spring Boot](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/)
 internally, Zipkin Server should not be considered a normal Spring Boot application.
 
-**Custom servers are not supported**
+## Custom servers are not supported
+
+By Custom servers we mean trying to use/embed `zipkin` as part of _your application_ (e.g. adding `zipkin-server` dependency to a Spring-boot application) instead of the packaged application we release.
```

Comment:
Maybe _your application_ > _an application you package_`? I guess most people that repackage with something custom still wouldn't consider it their own application, but it is their package